### PR TITLE
SIGINT-394 - refactor

### PR DESCRIPTION
**Pull request by lokeshsynopsys**

### CODE REVIEW

File: .github/workflows/all-scans.yml [hunk 1]
```diff
@@ -18,11 +18,11 @@ jobs:
   # This workflow contains a single job called "build"
   build:
     # The type of runner that the job will run on
-    runs-on: [self-hosted]
+    runs-on: [self-hosted, lokesh, macOS]
 
     env:
       MAC_BRIDGE_URL: "https://artifactory.internal.synopsys.com/artifactory/clops-local/clops.sig.synopsys.com/synopsys-action/0.1.61/ci-package-0.1.61-macosx.zip"
```
[the remark this answers is not in the document]
Comment (lokeshsynopsys) [reply .github/workflows/all-scans.yml]: updated

File: .github/workflows/blackduck.yml [hunk 2]
```diff
@@ -18,63 +18,27 @@ jobs:
...
         uses: ./
         with:
           blackduck_url: ${{ secrets.BLACKDUCK_URL }}
           blackduck_apiToken: ${{ secrets.BLACKDUCK_API_TOKEN_NEW }}
           blackduck_scan_full: true
-          blackduck_install_directory: /Users/kishori/synopsys-bridge
+          blackduck_install_directory: /root/synopsys-bridge
           blackduck_scan_failure_severities: "[\"BLOCKER\", \"CRITICAL\", \"TRIVIAL\"]"
-#          bridge_download_url: ${{ env.MAC_BRIDGE_URL }}
+          #bridge_download_url: ${{ env.LINUX_BRIDGE_URL }}
```
[the remark this answers is not in the document]
Comment (lokeshsynopsys) [reply .github/workflows/blackduck.yml]: updated

File: .github/workflows/polaris.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# This is a basic workflow to help you get started with Actions
```
[the remark this answers is not in the document]
Comment (lokeshsynopsys) [reply .github/workflows/polaris.yml:1]: as discussed, the config as code is in different folder

File: src/synopsys-action/input-data/coverity.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  coverity: CoverityConnect
+  project: ProjectData
+}
+
+export interface ProjectData {
+  repository?: {name: string}
+  branch?: {name: string}
+}
+
+export interface CoverityConnect {
```
[the remark this answers is not in the document]
Comment (lokeshsynopsys) [reply src/synopsys-action/input-data/coverity.ts:11]: As discussed, Coverity scan's input.json has an attribute called "coverity" itself.

**Review by tithiSynopsys — APPROVED** [review]

**Pull request merged**